### DISCUSSION TITLE
Should I add the drawio-exporter folder to git?

### DISCUSSION THREAD

**Discussion by aufbakanleitung** [LukeCarrier/mkdocs-drawio-exporter#27] [Q&A] [answered]
The exporter adds the pictures to a cache_dir called 'drawio-exporter' with a hash name such as `6b33167394941cdffef4856869d7401f55c6886e-0`. My initial reflex was to remove it from my version control, but I do add images. 
Would you say that it is better practise to leave it in or not?

**Comment by LukeCarrier** [comment, marked as answer]
Hey @aufbakanleitung, sorry for the delay :wave:

Good question! I think it depends:

- Since the content is derived from the diagrams, which are checked in to the repo, I'd generally recommend ignoring the `drawio-exporter` directory. 
- This will mean that clean checkouts will have to build all of the images the first time, which might be an issue for large sites with lots of diagrams. In this case, it might be worth checking it in.

I'll add something addressing this question to the README next time I work on the plugin 🙂

**Comment by aufbakanleitung** [comment]
Thanks @LukeCarrier,

I will ignore the directory for now.
